opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -3316,12 +3320,13 @@ const elUpdateConnectionsOfElement = async (documentId, documentBody) => {
 };
 export const elUpdateElement = async (instance) => {
   const esData = prepareElementForIndexing(instance);
+  validateDataBeforeIndexing(esData);
   const dataToReplace = R.dissoc('representative', esData);
   const replacePromise = elReplace(instance._index, instance.internal_id, { doc: dataToReplace });
   // If entity with a name, must update connections
   let connectionPromise = Promise.resolve();
   if (esData.name && isStixObject(instance.entity_type)) {
-    connectionPromise = elUpdateConnectionsOfElement(instance.internal_id, { name: esData.representative.main });
+    connectionPromise = elUpdateConnectionsOfElement(instance.internal_id, { name: esData.representative?.main ?? esData.name });
```

Comment:
unsure of that one.

Happens when using `stixCoreObjectImportPush`: representative is undefined